data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-utxo-creation-v1.0.xml

```diff
@@ -0,0 +1,211 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+
+    <changeSet author="R3.Corda" id="ledger-utxo-creation-v1.0">
+
+        <createTable tableName="utxo_transaction">
+            <column name="id" type="VARCHAR(160)">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="privacy_salt" type="VARBINARY(64)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="account_id" type="VARCHAR(64)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created" type="TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="utxo_transaction_signature">
+            <column name="transaction_id" type="VARCHAR(160)">
+                <constraints nullable="false" foreignKeyName="fk_utxo_transaction_signature" references="utxo_transaction(id)"/>
+            </column>
+            <column name="signature_idx" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="signature" type="VARBINARY(1048576)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="pub_key_hash" type="VARCHAR(160)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created" type="TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <addPrimaryKey columnNames="transaction_id, signature_idx"
+                       constraintName="utxo_transaction_signature_pkey"
+                       tableName="utxo_transaction_signature"/>
+
+        <createTable tableName="utxo_transaction_component">
+            <column name="transaction_id" type="VARCHAR(160)">
+                <constraints nullable="false" foreignKeyName="fk_utxo_transaction_component" references="utxo_transaction(id)"/>
+            </column>
+            <column name="group_idx" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="leaf_idx" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="data" type="VARBINARY(1048576)">
```

Comment:
That's an interesting one - this technically limits the size of states to arond 1MB. I am not sure this will hold in the long run...
How is this handled in Corda 4?